content/docs/authentication/social_authentication.md

```diff
@@ -76,7 +75,7 @@ OAuth providers require you to register a callback URL to handle the redirect re
 
 The callback URL must be registered with the OAuth service provider. For example: If you are using GitHub, you must log in to your GitHub account, [create a new app](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/creating-an-oauth-app) and define the callback URL using the GitHub interface.
 
-Also, you must register the same callback URL within the `config/ally.ts` file using the `callbackUrl` property.
+Also, you must register the same callback URL within the `config/ally.ts` file using the `callbackUrl` property. In many cases, you will want the callback URL to come from an Enviroment variable, or constructed based on a "public url" for your service.
```

Comment:
Whilst this isn't directly related, having language that says "hey, your Callback URL will be different between different environments" is something we should have.